pkg/operator/controllers/routefix/routefix.go

```diff
@@ -58,33 +106,129 @@ func (r *RouteFixReconciler) resources(ctx context.Context, cluster *arov1alpha1
 			},
 		},
 		scc,
+		&corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      kubeName,
+				Namespace: kubeNamespace,
+			},
+		},
+		&rbac.ClusterRoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: kubeName,
+			},
+			RoleRef: rbac.RoleRef{
+				APIGroup: "rbac.authorization.k8s.io",
+				Kind:     "ClusterRole",
+				Name:     "openshift-sd-controller",
+			},
+			Subjects: []rbac.Subject{
+				{
+					Kind:      "ServiceAccount",
+					Name:      kubeName,
+					Namespace: kubeNamespace,
+				},
+			},
+		},
+		&corev1.ConfigMap{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      kubeName,
+				Namespace: kubeNamespace,
+			},
+			Data: map[string]string{
+				"add_iptables.sh": shellScriptAddIptables,
+			},
+		},
 		&appsv1.DaemonSet{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      kubeName,
 				Namespace: kubeNamespace,
 			},
 			Spec: appsv1.DaemonSetSpec{
 				Selector: &metav1.LabelSelector{
-					MatchLabels: map[string]string{"app": "routefix"},
+					MatchLabels: map[string]string{"app": kubeName},
 				},
 				Template: corev1.PodTemplateSpec{
 					ObjectMeta: metav1.ObjectMeta{
-						Labels: map[string]string{"app": "routefix"},
+						Labels: map[string]string{"app": kubeName},
 					},
 					Spec: corev1.PodSpec{
 						Containers: []corev1.Container{
 							{
-								Name:  kubeName,
+								Name:  containerNameDrop,
 								Image: version.RouteFixImage(cluster.Spec.ACRDomain),
 								Args: []string{
 									"sh",
 									"-c",
-									shellScript,
+									shellScriptDrop,
 								},
 								// TODO: specify requests/limits
 								SecurityContext: &corev1.SecurityContext{
 									Privileged: to.BoolPtr(true),
 								},
+								Lifecycle: &corev1.Lifecycle{
+									PreStop: &corev1.Handler{
+										Exec: &corev1.ExecAction{
+											Command: []string{
+												"/bin/bash",
+												"-c",
+												"echo drop-icmp done",
+											},
+										},
+									},
+								},
+								VolumeMounts: []corev1.VolumeMount{
+									{
+										Name:      "host-slash",
+										MountPath: "/",
+										ReadOnly:  false,
+									},
+									{
+										Name:      "add-iptables",
+										MountPath: "/tmp/add_iptables.sh",
+										SubPath:   "add_iptables.sh",
+										ReadOnly:  false,
+									},
+								},
+								Resources: corev1.ResourceRequirements{
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resourceCPU,
+										corev1.ResourceMemory: resourceMemory,
+									},
+								},
+								Env: []corev1.EnvVar{
+									{
+										Name:  "OVN_KUBE_LOG_LEVEL",
+										Value: "4",
+									},
+									{
+										Name: "K8S_NODE",
+										ValueFrom: &corev1.EnvVarSource{
+											FieldRef: &corev1.ObjectFieldSelector{
+												FieldPath: "spec.nodeName",
+											},
+										},
+									},
+								},
```

Comment:
used?  If things aren't used, please remove them.

Reply:
`K8S_NODE` is referenced, verified we don't need `OVN_KUBE_LOG_LEVEL` but forgot to reap it.. it'll be gone in next update